modules/nf-core/gatk4/genomicsdbimport/main.nf

```diff
@@ -70,6 +70,7 @@ process GATK4_GENOMICSDBIMPORT {
 
     genomicsdb_command = "--genomicsdb-workspace-path ${prefix}"
     interval_command = interval_file ? "--intervals ${interval_file}" : "--intervals ${interval_value}"
+    updated_db = []
```

Comment:
Shouldn't this line be copied to the `script:` section too ? It feels like `updated_db` should also have a default value there

Reply:
Done!